daemon/analytics.c

```diff
@@ -411,13 +411,13 @@ void analytics_misc(void)
 {
 #ifdef ENABLE_ACLK
     analytics_set_data(&analytics_data.netdata_host_cloud_available, "true");
-#ifdef ACLK_NG
-    analytics_set_data_str(&analytics_data.netdata_host_aclk_implementation, "Next Generation");
-#else
-    analytics_set_data_str(&analytics_data.netdata_host_aclk_implementation, "legacy");
-#endif
+    if (aclk_ng)
```

Comment:
will this always come after `aclk_starter`?

Reply:
Yes, it will actually be called 2 minutes after the agent start (it's the amount we let to make sure things have settled a bit...). I assume it should be enough?